src/hogwarts.java

```diff
@@ -1,3 +1,59 @@
 public class hogwarts {
```

Comment:
Классы называются с большой буквы. 
```suggestion
public class Hogwarts {
```